egg/core/language_analysis.py

```diff
@@ -205,8 +205,10 @@ def compute_topsim(
     def print_message(self, logs: Interaction, mode: str, epoch: int) -> None:
         messages = logs.message.argmax(dim=-1) if self.is_gumbel else logs.message
         messages = [msg.tolist() for msg in messages]
+        sender_input = torch.flatten(logs.sender_input, start_dim=1)
 
-        topsim = self.compute_topsim(logs.sender_input, messages)
+        topsim = self.compute_topsim(sender_input, messages,
+                                     self.sender_input_distance_fn, self.message_distance_fn)
```

Comment:
Any chance this could be formatted to be black compliant?
Generally, we try to enforce the a black coding style.
You could simply put all args in one line or one per line if the line exceeds the limit (which iirc should be 80)